ipaserver/install/ca.py

```diff
@@ -231,7 +227,7 @@ def hsm_validator(token_name, token_library, token_password):
         # It may fail if p11-kit has already registered the library, that's
         # ok.
         ipautil.run(command, stdin='\n', cwd=tempnssdb.secdir,
-                    runas=pkiuser, suplementary_groups=[pkigroup],
+                    runas=pkiuser, suplementary_groups=[group_list],
```

Comment:
`os.getgrouplist()` already returns a list, so you don't need to put it into a list, e.g. use `suplementary_groups=group_list`

Reply:
Bah, I had it right then broke it. Fixed now.